omega/omega-transaction/src/main/java/org/apache/servicecomb/saga/omega/transaction/SagaStartAnnotationProcessor.java

```diff
@@ -19,25 +19,40 @@
 
 import org.apache.servicecomb.saga.omega.context.OmegaContext;
 
-public class SagaStartAnnotationProcessor {
+class SagaStartAnnotationProcessor implements EventAwareInterceptor {
 
   private final OmegaContext omegaContext;
-
   private final MessageSender sender;
 
   SagaStartAnnotationProcessor(OmegaContext omegaContext, MessageSender sender) {
     this.omegaContext = omegaContext;
     this.sender = sender;
   }
 
-  void preIntercept() {
-    String globalTxId = omegaContext.newGlobalTxId();
+  @Override
+  public void preIntercept(String parentTxId, String compensationMethod, Object... message) {
+    String globalTxId = globalTxId();
     // reuse the globalTxId as localTxId to differ localTxId in SagaStartedEvent and the first TxStartedEvent
     sender.send(new SagaStartedEvent(globalTxId, globalTxId));
```

Comment:
move the above to the new method, the use omegaContext.globalTxId(), omegaContext.localTxId() here is better?